Tests/JetworkingTests/ClientTests/ClientTests.swift

```diff
@@ -242,6 +242,59 @@ final class ClientTests: XCTestCase {
         
         waitForExpectations(timeout: 140.0, handler: nil)
     }
+    func testUploadFile() {
+        let client = Client(configuration: makeDefaultClientConfiguration())
+        let expectation = self.expectation(description: "Wait for upload")
+        
+        let url = URL(string: "https://catbox.moe/user/api.php")!
+        let documentsUrl: URL = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first!
+        let filename = "avatar.png"
+        let fileURL = documentsUrl.appendingPathComponent(filename)
+        client.upload(
+            url: url,
+            fileURL: fileURL,
+            progressHandler: { (bytesSent, bytesExpectedToSend) in
+                let progress = Float(bytesSent) / Float(bytesExpectedToSend)
```

Comment:
You could validate that your bytes sent is always less then `bytesExpectedToSend`

Reply:
Done.